api/src/pcapi/routes/backoffice/pivots/blueprint.py

```diff
@@ -73,9 +77,21 @@ def create_pivot(name: str) -> utils.BackofficeResponse:
         flash(utils.build_form_error_msg(form), "warning")
         return redirect(url_for(".get_pivots", active_tab=name), code=303)
 
+    venue = offerer_models.Venue.query.filter_by(id=form.venue_id.data[0]).one_or_none()
+    if not venue:
+        flash("Lieu inexistant", "warning")
```

Comment:
```suggestion
        flash(Markup("Le lieu id={venue_id} n'existe pas").format(venue_id=venue_id), "warning")
```
Un peu d'uniformité, c'est important